donor-registry/src/app/forms/forms.component.ts

```diff
@@ -141,9 +141,21 @@ export class FormsComponent implements OnInit {
   flag: boolean = true;
   routeNew: string;
   tempUrl: string;
+  organCheckbox: boolean = true;
 
 
   ngAfterViewChecked() {
+    if (!this.organCheckbox) {
+      if (!this.model['pledgeDetails']['organs'] && !this.model['pledgeDetails']['tissues']) {
+        this.removeElement("oterrormsg");
+        this.addElement('Organs_and_Tissues_to_Pledge', 'Please select atleast one organs or tissues', 'oterrormsg')
```

Comment:
@suvarnakale Add this string into language constant file
`'Please select atleast one organs or tissues'`

Reply:
okay